pantheon-bundle/frontend/src/app/productDetails.test.tsx

```diff
@@ -1,137 +1,112 @@
 import React from "react"
-import { ProductDetails, IProps } from "@app/productDetails"
 import "@app/fetchMock"
-
 import { mount, shallow } from "enzyme"
 import { Breadcrumb, Button, Form, FormGroup, Level, LevelItem, TextContent, TextInput, Text } from "@patternfly/react-core"
-import renderer from "react-test-renderer"
-import sinon from "sinon"
-
-const props = {
-  productName: "Red Hat Enterprise Linux"
+import { ProductDetails } from "@app/productDetails"
+import { ProductContext, IProduct } from "./contexts/ProductContext"
+
+const allProducts: IProduct[] = [{
+  description: "descrip",
+  isOpen: false,
+  ["jcr:created"]: 1605556710885,
+  ['jcr:createdBy']: "admin",
+  ['jcr:lastModified']: 1605556710886,
+  ['jcr:lastModifiedBy']: "admin",
+  ['jcr:primaryType']: "pant:product",
+  ['jcr:uuid']: "5df8d913-79b9-42fd-a17b-ffc917add446",
+  locale: "en-US",
+  name: "product 11/16",
+  ['sling:resourceType']: "pantheon/product",
+  urlFragment: "product_url",
+}, {
+  description: "test descrip",
+  isOpen: false,
+  ['jcr:created']: 1605621399917,
+  ['jcr:createdBy']: "admin",
+  ['jcr:lastModified']: 1605621399918,
+  ['jcr:lastModifiedBy']: "admin",
+  ['jcr:primaryType']: "pant:product",
+  ['jcr:uuid']: "242d4187-d2f4-4df5-917b-e09bf4ff45e9",
+  locale: "en-US",
+  name: "test 11/17",
+  ['sling:resourceType']: "pantheon/product",
+  urlFragment: "testurl",
+}];
+
+const props =
+{
+  match:
+  {
+    params: {
+      id: '5df8d913-79b9-42fd-a17b-ffc917add446'
```

Comment:
What's this ID, not sure if it's debug code? If it isn't, maybe a comment to explain why there's a hardcoded id?